reqs/tests/import_reqs_tests.py

```diff
@@ -140,3 +140,101 @@ def test_repeat_row():
     processor2.add(row)
     assert Requirement.objects.count() == 1
     assert Requirement.objects.first().issuing_body == 'body2'
+
+
+@pytest.mark.django_db
+def test_no_matching_policy_type():
+    """If the policy type isn't found, raise an exception"""
+    row = {'policyNumber': '123', 'policyTitle': 'title 1',
+           'uriPolicyId': 'http://example.com/a', 'ombPolicyId': 'policy',
+           'policyType': 'IAmAnOutlier', 'policyIssuanceYear': '12/20/2001',
+           'policySunset': 'NA'}
+    policy_proc = import_reqs.PolicyProcessor()
+    with pytest.raises(ValueError):
+        policy_proc.from_row(row)
+
+
+@pytest.mark.django_db
+def test_varying_headers():
+    """
+    Test that we can handle a variety of names for fields (in this case, the
+    OMB Policy ID field).
+    """
+    numbers = iter(range(1, 200))
+    row = {'policyNumber': '123', 'policyTitle': 'title 1',
+           'uriPolicyId': 'http://example.com/a',
+           'policyType': 'Guidance', 'policyIssuanceYear': '12/20/2001',
+           'policySunset': 'NA'}
+    for key in ("ombPolicyId", "ombPolicyID"):
```

Comment:
This _might_ be a good use case for `pytest.mark.parametrize` -- it'd generate multiple tests which would tell us exactly which key name was wonky.